src/ops/call_contract.rs

```diff
@@ -50,6 +50,13 @@ impl<S: Resolver<H>, H: ByteHash> AbiCall<S, H> for CallContract {
         }
 
         // Perform the call
-        context.call(target, argument_ptr, argument_len, return_ptr, return_len)
+        context.call(
+            target,
+            0,
```

Comment:
Does this mean the opcode `0` is always called, no matter what is passed?

Reply:
For the inter-contract call at the moment yes: the opcode is passed only when you call the contract from outside another contract, see `state.rs`
I want to double check with @krl  the approach before extend it; at that point I either add a `call_operation` method as well in the `ops` or we refactor the all `contract`'s call to have opcode by default as argument (that would break the current API also, if a contract has only one operation, doesn't need enums).

Reply:
Since the question is now discussed, I will update the inter-contract call, but I will do in a follow up issue in order to test it properly as well.